src/main/java/com/greenfox/tribesoflagopus/backend/model/entity/Kingdom.java

```diff
@@ -75,19 +73,12 @@ public void removeBuilding(Building building) {
     this.buildings.remove(building);
   }
 
-  @OneToOne(mappedBy = "kingdom", cascade = CascadeType.ALL, orphanRemoval = true)
-  private Location location;
-
-  @Setter(AccessLevel.NONE)
-  @OneToMany(mappedBy = "kingdom", cascade = CascadeType.ALL, orphanRemoval = true)
-  private List<Resource> resources;
-
-  public void addResources(Resource resource) {
+  public void addResource(Resource resource) {
```

Comment:
OK, but if you are using this, it's a good idea to delete your setter for your `resource` field to avoid errors coming from misunderstanding.